apps/src/sites/studio/pages/print_certificates/show.js

```diff
@@ -1,6 +1,10 @@
 import analyticsReporter from '@cdo/apps/lib/util/AnalyticsReporter';
 import {EVENTS} from '@cdo/apps/lib/util/AnalyticsConstants';
+import {getStore} from '@cdo/apps/redux';
 
 $(() => {
-  analyticsReporter.sendEvent(EVENTS.CERTIFICATE_PRINT_PAGE_VISITED);
+  const isTeacher = getStore().getState().currentUser.userType === 'teacher';
+  if (isTeacher) {
+    analyticsReporter.sendEvent(EVENTS.CERTIFICATE_PRINT_PAGE_VISITED);
+  }
```

Comment:
Learning Q: is this how you add Amplitude events on studio.code.org? (new logic notwithstanding)

Reply:
yuuup it's v simple on dashboard!